ovos_utils/ovos_service_api.py

```diff
@@ -172,6 +172,7 @@ def search_movie(self, query):
         r = requests.post(url, data=reqdata, headers=self.headers)
         return r.json()
 
+
```

Comment:
can we mark this whole file for deprecation too? ovos-backend-client should be used now